application/templates/pages/cookies.html

```diff
@@ -39,7 +39,7 @@ <h3 class="govuk-heading-m" id="our-cookie-banner">Our cookie banner</h3>
           </tr>
         </tbody>
       </table>
-      <h3 class="govuk-heading-m" id="google-analytics-cookies">Google Analytics cookies</h3>
+      {# <h3 class="govuk-heading-m" id="google-analytics-cookies">Google Analytics cookies</h3>
```

Comment:
Wasn't sure if to comment this out or remove? What is git/code best practice. Somebody has to remember/know that it was once there if we need it back if we remove?